pkg/doc/bbs/bbs12381g2pub/internal/kilic/bls12-381/fp_fallback.go

```diff
@@ -0,0 +1,416 @@
+// Code generated by goff (v0.3.5) DO NOT EDIT
+
```

Comment:
should still add license header like other files.

Reply:
OK

Reply:
^^^